release_notes/ose_3_2_release_notes.adoc

```diff
@@ -0,0 +1,736 @@
+= OpenShift Enterprise 3.2 Release Notes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+OpenShift Enterprise by Red Hat is a Platform as a Service (PaaS) that provides
+developers and IT organizations with a cloud application platform for deploying
+new applications on secure, scalable resources with minimal configuration and
+management overhead. OpenShift Enterprise supports a wide selection of
+programming languages and frameworks, such as Java, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Google Kubernetes, OpenShift Enterprise
+provides a secure and scalable multi-tenant operating system for today’s
+enterprise-class applications, while providing integrated application runtimes
+and libraries. OpenShift Enterprise brings the OpenShift PaaS platform to
+customer data centers, enabling organizations to implement a private PaaS that
+meets security, privacy, compliance, and governance requirements.
+
+[[ose-32-new-features-and-enhancements]]
+== New Features and Enhancements
+
+OpenShift Enterprise version 3.2 is now available. Ensure that you follow the
+instructions on upgrading your OpenShift cluster properly, including steps
+specific to this release.
+
+[IMPORTANT]
+====
+For any release, always review link:../install_config/upgrading/index.html[Installation
+and Configuration] for instructions on
+link:../install_config/upgrading/index.html[upgrading your OpenShift cluster] properly,
+including any additional steps that may be required for a specific release.
+====
```

Comment:
Hmmm, probably at the time? Came over from the 3.1 version of this topic. Looks like I got link-happy. Might drop the 2nd one.